UKHomeOffice/cerberus-service
# COP-11683 - Issue target note missing

## Description
This PR simply adds a delay for when a promise is fulfilled for when submitting a target information sheet.

## To Test
- Pull & run against dev
- Click on `Issue target` and fill out the form (RoRo)
- Submit the form.
- On form submission, the activity log should display a note recording the target has been issued.
